### ISSUE TITLE
Add icon for when media doesn't have alt text.

### ISSUE THREAD

**Issue by egel557** [elk-zone/elk#1570]
There's already an icon for when a media has alt text, but I think it would also be useful to have one for when a media doesn't. It's often more important to know when a media doesn't have alt text than when it does. Particularly, when you're trying to avoid boosting posts without it.

Megalodon already does something similar. It shows an exclamation mark icon with a red background, making it stand out.

<img src="https://user-images.githubusercontent.com/58113537/216336358-ce03e32c-ee67-45e9-923e-fd52ffbed917.jpg" width="300" />

This could toggleable as well for people who don't want it.

**Comment by ZMYaro** [comment]
I brought this up in #1410, but I think a straightforward “NO ALT” badge would be better since an “!” icon could mean one of several potential issues with an image.

**Comment by egel557** [comment]
I agree, that definitely sounds like a good approach.

**Comment by boehs** [comment]
Duplicate of #1552

**Comment by ZMYaro** [comment]
@boehs Not exactly.  #1552 is about the composer, while this seems to be about the feed view.  I do think we could rename #1410 to be more comprehensively about rethinking feed view alt text indicators and then close this as a duplicate.

**Comment by boehs** [comment]
![image](https://user-images.githubusercontent.com/51836263/216478411-bf72bca7-abd5-496f-8039-e398bc81a048.png)

**Comment by ZMYaro** [comment]
I don't think it necessarily needs as extensive a popover, if that is @patak-dev's concern, but I think we should at least have the “NO ALT” badge (and as I said on #1410, I support removing the “ALT” badge).
